web/docs/guides/auth.mdx

```diff
@@ -27,17 +27,22 @@ We currently support the following OAuth providers:
 - Azure
 - Facebook
 - Bitbucket
+- Twitter
+- Apple
 
 ![OAuth Logins.](/img/supabase-oauth-logins.png)
 
 You can enable providers by navigating to Authentication > Settings > External OAuth Providers and inputting your `Client ID` and `Secret` for each.
 
 To fetch these, you need to:
 
-1. Generate `Client ID` and `Secret` ([Google](https://console.developers.google.com/apis/credentials), [GitHub](https://github.com/settings/applications/new), [GitLab](https://gitlab.com/oauth/applications), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud/)).
+1. Generate `Client ID` and `Secret` ([Google](https://console.developers.google.com/apis/credentials), [GitHub](https://github.com/settings/applications/new), [GitLab](https://gitlab.com/oauth/applications), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud/), [Apple](https://developer.apple.com/account/ios/certificate), [Twitter](https://developer.twitter.com/en/portal/projects-and-apps) ).
 2. Enter Authorized Redirect URI: `https://<your-project>.supabase.co/auth/v1/callback` on provider dashboard.
 
-### Row Level Security
+### Enable Sign-In with Apple
+Currently, we do not generate the `client_secret` the Apple requires, but here's a [useful guide](https://medium.com/identity-beyond-borders/how-to-configure-sign-in-with-apple-77c61e336003) on how you can generate it on your end!
```

Comment:
> the Apple requires

sp: that Apple requires.


I think we can even just put "See [this guide] to generate an Apple `client_id`." beside the "Apple" bullet point above.

Reply:
oops thanks for spotting that! sounds good, will make the change!